app/sprinkles/account/src/Database/Migrations/v430/UpdateGroupsTable.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * UserFrosting (http://www.userfrosting.com)
+ *
+ * @link      https://github.com/userfrosting/UserFrosting
+ * @copyright Copyright (c) 2019 Alexander Weissman
+ * @license   https://github.com/userfrosting/UserFrosting/blob/master/LICENSE.md (MIT License)
+ */
+
+namespace UserFrosting\Sprinkle\Account\Database\Migrations\v430;
+
+use Illuminate\Database\Schema\Blueprint;
+use UserFrosting\Sprinkle\Core\Database\Migration;
+
+/**
+ * Groups table migration
+ * Changes the `icon` column property of `default` to NULL to align with new Font Awesome 5 tag convention.
+ * Version 4.3.0
+ *
+ * See https://laravel.com/docs/5.4/migrations#tables
+ * @author Alex Weissman (https://alexanderweissman.com)
+ */
+class UpdateGroupsTable extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public static $dependencies = [
+        '\UserFrosting\Sprinkle\Account\Database\Migrations\v400\GroupsTable'
+    ];
+
+    /**
+     * {@inheritdoc}
+     */
+    public function up()
+    {
+        if ($this->schema->hasTable('groups')) {
+            $this->schema->table('groups', function (Blueprint $table) {
+                $table->string('icon', 100)->default('NULL')->change();
```

Comment:
Should it be null, or an empty string? 🤔

Reply:
I went with NULL based off this post: https://stackoverflow.com/questions/38351498/remove-default-in-migration

I also compared to the other columns in the table that did not have a default set when the migration ran, and they all showed `NULL` for the `Default` property. In mysql here is how the results of my testing:
![image](https://user-images.githubusercontent.com/33728190/55689605-5afa7880-5954-11e9-8854-a5dbec851f9c.png)

Reply:
Should be possible to do an export of the create table logic to see if the default value was truly reversed (the cleaner the better, leftovers attract edge cases).

Assuming I remember, I'll take a look at what the DDL ends up being.

Reply:
Here is how default migration looks
![userfrosting groups default](https://user-images.githubusercontent.com/33728190/56537662-144d7680-652f-11e9-8549-4fe2293dc913.png)

Here is the new migration `up`
![UP](https://user-images.githubusercontent.com/33728190/56537669-16afd080-652f-11e9-8dd7-7b148ff026ff.png)

New migration `down`
![DOWN](https://user-images.githubusercontent.com/33728190/56537672-18799400-652f-11e9-8c03-ae1385fe6428.png)

Reply:
I think the posted DDL is wrong. Or its mislabeled? I'll still have to look at this myself, but it works as a good reminder none-the-less, so thanks.

Reply:
I am curious about what is wrong? 

Other than `('fas fa-user')` in [this line](https://github.com/amosfolz/UserFrosting/blob/fa5/app/sprinkles/account/src/Database/Migrations/v430/UpdateGroupsTable.php#L50), the down method returns the table to the 'original' state. I was not certain how to handle this but I thought it would be better to set the default to use the new `fas` convention rather than the `fa`.

I relabeled my pictures above that might help clear up any confusion.